apps/HelloAndroidCamera2/jni/HalideFilters.cpp

```diff
@@ -35,6 +34,29 @@ extern "C" void halide_print(void *, const char *msg) {
 
 extern "C" {
 
+bool assertEqualExtents(YuvBufferT *src, YuvBufferT *dst) {
```

Comment:
Rename back to equalExtents(). assert is weird. I would also prefer const ref arguments over pointer.

(Don't feel that strongly) Having the LOGE inside is also a bit weird. I see that having the message duplicated outside is also annoying. Maybe make a separate function?